### ISSUE TITLE
Testing infrastructure

### ISSUE THREAD

**Issue by aimalz** [aimalz/proclam#12]
Before the modules can get off the ground, we need a script that tests arbitrary combinations of test conditions.  It should:

1. read in a config file
2. make true class assignments
3. import the classifier
4. make predicted classes
5. import the metric
6. evaluate the metric
7. make informative plots (???) 

Once #8 produces the function for item 2, this should be pretty easy to write if there's at least one classifier module and at least one metric module.  And when it's finished, we can run some reasonable combinations (without the reasonable restriction, there could be a lot!) in parallel with another script.

**Comment by aimalz** [comment]
At this point, I don't think it's necessary to do file I/O, and the informative plots can be postponed.  #15 otherwise addresses everything.
